app/models/evss_claim.rb

```diff
@@ -3,5 +3,21 @@
 require 'evss/documents_service'
 
 class EVSSClaim < ApplicationRecord
-  scope :for_user, ->(user) { where(user_uuid: user.uuid) }
+  belongs_to :user_account, dependent: nil, optional: true
+
+  def self.for_user(user)
+    claim_for_user_uuid(user.uuid).or(claim_for_user_account(user.user_account))
+  end
+
+  def self.claim_for_user_uuid(user_uuid)
+    return EVSSClaim.none unless user_uuid
+
+    where(user_uuid: user_uuid)
+  end
+
+  def self.claim_for_user_account(user_account)
+    return EVSSClaim.none unless user_account
+
+    where(user_account: user.user_account)
```

Comment:
@bosawt Is this supposed to be:
```ruby
where(user_account: user_account)
```
We are starting to see this error in staging as of the last hour or so: http://sentry.vfs.va.gov/organizations/vsp/issues/173747

Reply:
@jerekshoe I just merged a fix into staging now

Reply:
Ok cool, thanks for letting me know!